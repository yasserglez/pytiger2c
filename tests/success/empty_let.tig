let
in
    print("This Tiger program has an empty let.\n")
end
