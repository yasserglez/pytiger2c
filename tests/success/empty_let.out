This Tiger program has an empty let.
